test_utils/scripts/get_target_packages.py

```diff
@@ -44,6 +44,11 @@
     'speech': {'storage'},
     'vision': {'storage'},
 }
+REVERSE_DEPS = {}
+for pkg, deps in PKG_DEPENDENCIES.items():
+    for dep in deps:
+        REVERSE_DEPS.setdefault(dep, set()).add(pkg)
+del pkg, deps
```

Comment:
To push that further:  ISTM that you could just as well move the whole kit-and-caboodle (`PKG_DEPENDENCIES`, `REVERSE_DEPS`) inside `get_changed_packages()`, which is their only customer.